### PR TITLE
Edit all occurences of "watchface" to "watch face"

### PR DESCRIPTION
**Pull request by mashuptwice**
Exact opposite of #1303 .

Before:
```bash
╰─$ grep -r "Watchface\|watchface"                                                                                                                                                                           130 ↵
doc/MemoryAnalysis.md:Using this technique, I was able to trace all malloc calls at boot (boot -> digital watchface):
doc/InfiniTimeVision.md:- Personalization is achieved through custom watchfaces.
doc/InfiniTimeVision.md:- Capability to sideload apps and watchfaces
doc/gettingStarted/updating-software.md:You can check the InfiniTime version by first swiping right on the watchface to open quick settings, tapping the cogwheel to open settings, swipe up until you find an entry named "About" and tap on it.
doc/gettingStarted/updating-software.md:- From the watchface, swipe **right** to display the *quick settings menu*
doc/gettingStarted/gettingStarted-1.0.md:By default, InfiniTime starts on the digital watchface. It'll probably display the epoch time (1 Jan 1970, 00:00).
doc/gettingStarted/gettingStarted-1.0.md:![Digital watch face](ui/watchface.jpg)
src/displayapp/screens/WatchFacePineTimeStyle.cpp: * PineTimeStyle watchface for Infinitime created by Kieran Cawthray
src/components/ble/weather/WeatherData.h:       * As direction can fluctuate wildly and some watchfaces might wish to display it nicely,
src/components/ble/weather/WeatherData.h:       * Location info can be for some kind of map watchface
src/components/ble/weather/WeatherData.h:       * this allows watchface or watchapp makers to generate accurate alerts and graphics
```
After:
```bash
╰─$ grep -ri "watch face"                                                                                                                                                                                    130 ↵
doc/MemoryAnalysis.md:Using this technique, I was able to trace all malloc calls at boot (boot -> digital watch face):
doc/InfiniTimeVision.md:- Personalization is achieved through custom watch faces.
doc/InfiniTimeVision.md:- Capability to sideload apps and watch faces
doc/gettingStarted/updating-software.md:You can check the InfiniTime version by first swiping right on the watch face to open quick settings, tapping the cogwheel to open settings, swipe up until you find an entry named "About" and tap on it.
doc/gettingStarted/updating-software.md:- From the watch face, swipe **right** to display the *quick settings menu*
doc/gettingStarted/gettingStarted-1.0.md:By default, InfiniTime starts on the digital watch face. It'll probably display the epoch time (1 Jan 1970, 00:00).
doc/gettingStarted/gettingStarted-1.0.md:### Digital watch face
doc/gettingStarted/gettingStarted-1.0.md:![Digital watch face](ui/watch face.jpg)
doc/gettingStarted/gettingStarted-1.0.md:This is what the default digital watch face looks like. You can change watch faces in the settings.
doc/gettingStarted/gettingStarted-1.0.md:- You can hold the button for a short time to return to the watch face. (version >1.7.0)
src/CMakeLists.txt:        ## Watch faces
src/displayapp/screens/WatchFacePineTimeStyle.cpp: * PineTimeStyle watch face for Infinitime created by Kieran Cawthray
src/displayapp/screens/settings/Settings.h:          {Symbols::home, "Watch face", Apps::SettingWatchFace},
src/displayapp/screens/settings/SettingWatchFace.cpp:  lv_label_set_text_static(title, "Watch face");
src/components/ble/weather/WeatherData.h:       * As direction can fluctuate wildly and some watch faces might wish to display it nicely,
src/components/ble/weather/WeatherData.h:       * Location info can be for some kind of map watch face
src/components/ble/weather/WeatherData.h:       * this allows watch face or watchapp makers to generate accurate alerts and graphics
```

### CODE REVIEW

File: doc/gettingStarted/gettingStarted-1.0.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ InfiniTime doesn't handle daylight savings automatically, so make sure to set th
 
 ### Digital watch face
 
-![Digital watch face](ui/watchface.jpg)
+![Digital watch face](ui/watch face.jpg)
```
Comment (NeroBurner) [inline doc/gettingStarted/gettingStarted-1.0.md:25]: I think the jpg filename wasn't renamed (and shouldn't be ;)

```suggestion
![Digital watch face](ui/watchface.jpg)
```

Comment (mashuptwice) [reply doc/gettingStarted/gettingStarted-1.0.md:25]: Absolutely! One of the caveats of regex..

Comment (NeroBurner) [reply doc/gettingStarted/gettingStarted-1.0.md:25]: maybe you want to revert this line as well?

**Review by Avamander — APPROVED** [review]
This is small enough of a change that we could merge this soon

**Review by NeroBurner — APPROVED** [review]
I agree, let's merge this one as it feels like the "more right" one

**Pull request merged**